timed/reports/views.py

```diff
@@ -296,7 +308,15 @@ def _generate_workreport_name(self, from_date, today, project):
             self._clean_filename(project.name),
         )
 
-    def _create_workreport(self, from_date, to_date, today, project, reports, user):  # noqa: PLR0913
+    def _create_workreport(  # noqa: PLR0913
+        self,
+        from_date: date,
+        to_date: date,
+        today: date,
```

Comment:
Passing today seems like something we'd want to take a closer look at and refactor in a diff PR.

Reply:
yup: #1074